fetch-todolist/src/component/Home.js

```diff
@@ -29,29 +46,28 @@ const Home = () => {
   );
 
   const changeStatus = (check, id) => {
-    const updatedStatusArray = data.map((obj) => {
+    const object = data.find((obj) => {
       if (obj.id === id) {
+        console.log("id", id, "check ", check);
         obj.status = check;
       }
-      return obj;
     });
-    setTodos(updatedStatusArray);
+    dispatch({ type: ACTIONS.SET_TODOS, payload: data });
```

Comment:
can we write action dispatcher ? which means a function that will return a payload: { type: ACTIONS.SET_TODOS, payload: data }

Reply:
yes we can add a function which will return the action and give it's address to dispatch

Reply:
For now I just added a toggleDeleteFunction for the example, I've followed the same in subsequent PR's for React Redux.

Reply:
toggleDeleteBlock() Function*